cypress/e2e/smoke/Wallets/fiatDeposit.cy.js

```diff
@@ -19,6 +19,7 @@ describe('QATEST-98805 - Fiat deposit iframe access', () => {
     cy.log('Access Fiat Deposit Iframe')
     cy.c_visitResponsive('/', 'small')
     cy.contains('Wallet', { timeout: 10000 }).should('exist')
+    cy.c_skipPasskeysV2()
```

Comment:
better to run before the asssertion no?
and can we not use findByText instead of contains if it is partial text you are trying to check then can use 
cy.findByText(/wallets/) or cy.findAllByText(/wallets/) if multiple elements

```suggestion
    cy.c_visitResponsive('/', 'small')
        cy.c_skipPasskeysV2()
    cy.contains('Wallet', { timeout: 10000 }).should('exist')
```

Reply:
this is still the same.

Reply:
this is missed again @afsaneh-rezaeinejad-deriv